backend/app/forgot-password.py

```
@@ -0,0 +1,9 @@
+# In your database models file (e.g., models.py)
+import datetime
```

Ruff (0.12.2)

`datetime` imported but unused

Remove unused import: `datetime`

(F401)